tests/users.rs

```diff
@@ -15,7 +15,13 @@ mod get_users_me {
 
         let response_user: User = conn.response_json().await;
 
-        assert_eq!(user, response_user);
+        assert_eq!(user.email, response_user.email);
+        assert_eq!(user.email_verified, response_user.email_verified);
+        assert_eq!(user.name, response_user.name);
+        assert_eq!(user.nickname, response_user.nickname);
+        assert_eq!(user.picture, response_user.picture);
+        assert_eq!(user.sub, response_user.sub);
+        assert_eq!(user.admin, response_user.admin);
```

Comment:
This is because different codepaths generate a `User` on-demand using `User::for_integration_testing`, and so they'll get `User`s with different `updated_at` values since `User::for_integration_testing` calls `OffsetDateTime::now_utc`. We could just change `User::for_integration_testing` to use a fixed `updated_at` and go back to using `User::eq`.